lib/mayaUsd/fileio/primUpdaterManager.cpp

```diff
@@ -1022,6 +1023,15 @@ bool PrimUpdaterManager::discardPrimEdits(const Ufe::Path& pulledPath)
         updater->discardEdits();
     }
 
+    // Reset the auto-edit when discarding the edit.
```

Comment:
Essentially, we are checking for a particular node type here --- and I don't think that's a good idea.  I would really hope we can implement an override of the discardEdits() virtual on MayaReferenceUpdater and do the work there.